superset-frontend/src/chart/Chart.jsx

```diff
@@ -220,6 +220,7 @@ class Chart extends React.PureComponent {
 
     return (
       <ChartErrorMessage
+        key={chartId}
```

Comment:
was this needed to fix the issue too? i'm surprised that setting the key to the id would cause a rerender since i wouldn't expect the id to ever change

Reply:
This is a bycatch. Just for fixing a React warning about missing key in loops.

Reply:
ah thanks, i didn't see a loop anywhere. does this mean the `Styles` above needs a key too?

Reply:
The loop is here: https://github.com/apache/superset/blob/157843be087f2f23a0df7c9efad8f2603150f9dc/superset-frontend/src/chart/Chart.jsx#L252

Good catch on the styled component above!